seqr/fixtures/users.json

```diff
@@ -119,6 +135,42 @@
         "user_permissions": []
     }
 },
+{
+    "model": "auth.user",
+    "pk": 16,
+    "fields": {
+        "password": "pbkdf2_sha256$30000$y85kZgvhQ539$jrEC3L1IhCezUx3Itp+14w/T7U6u5XUxtpBZXKv8eh4=",
+        "last_login": null,
+        "is_superuser": false,
+        "username": "test_data_manager",
+        "first_name": "Test Data Manager",
+        "last_name": "",
+        "email": "test_data_manager@test.com",
+        "is_staff": true,
```

Comment:
no, the built in django user model we use only has 2 possible permission flags: is_staff and is_superuser. We are using is_staff to correspond to the data manager role now as its the easiest way to store that privilege rather than creating another user group to maintain